WordPress/src/main/java/org/wordpress/android/ui/jetpack/common/viewholders/IconViewHolder.kt

```diff
@@ -10,9 +10,9 @@ import org.wordpress.android.util.image.ImageManager
 class IconViewHolder(
```

Comment:
Idea (💡): I know this is not this PRs change, but since this renaming is done now, maybe consider renaming the `JetpackViewHolder` child classes to hold the `Jetpack` name in them and avoid having them being generic. Same logic that we discussed on the `Backup` naming conventions, for searchability purposes. So, I propose:
- from `IconViewHolder` -> to `JetpackIconViewHolder` or `IconJetpackViewHolder`
- from `HeaderViewHolder` -> to `JetpackHeaderViewHolder` or `HeaderJetpackViewHolder`
- from `DescriptionViewHolder` -> to `JetpackDescriptionViewHolder` or `DescriptionJetpackViewHolder`
- from `ActionButtonViewHolder` -> to `JetpackButtonViewHolder` or `ButtonJetpackViewHolder`

Note that I removed the `Action` from the `ActionButtonViewHolder` to match that with the `jetpack_list_button_item` layout file, which doesn't contains it. Else you can add the `action` to the layout file, either way works depending how much you want the `Action` word in them.

Reply:
Ahh, that is a GOOD idea!! Going to do that now! @ashiagr FYI